nestkernel/secondary_event.h

```diff
@@ -23,6 +23,8 @@
 #ifndef SECONDARY_EVENT_H
 #define SECONDARY_EVENT_H
 
+#include <set>
```

Comment:
For consistency with other files
```suggestion
// C++ includes:
#include <set>
```